docs/src/pages/guides/FHIRBulkOperations.md

```diff
@@ -0,0 +1,193 @@
+    "resourceType": "Parameters",
+    "id": "e33a6a4e-29b5-4f62-a3e9-8d09f50ae54d",
+    "parameter": [
+        {
+            "name": "_outputFormat",
+            "valueString": "application/fhir+ndjson"
+        },
+        {
+            "name": "_since",
+            "valueInstant": "2019-01-01T08:21:26.94-04:00"
+        },
+        {
+            "name": "_type",
+            "valueString": "Observation,Condition"
+        }
+    ]
+}'
+```
+
+## Import Operation: $import
+
+The `$import` operation is System operation are invoked at `[base]/$import`.  The Import Operation uses a custom crafted OperationDefinition [link](https://github.com/IBM/FHIR/blob/master/fhir-operation-bulkdata/src/main/resources/import.json), which follows the proposal from [Smart-on-FHIR: import.md](https://github.com/smart-on-fhir/bulk-import/blob/master/import.md).
+
+### **$import: Create a Bulk Data Request**
+To create an import request, the IBM FHIR Server requires the body fields of the request object to be a FHIR Resource `Parameters` JSON Object.  The request must be posted to the server using `POST`. Each request is limited to a single resource type in each imported or referenced file.
+
+The IBM FHIR Server limits the number of inputs per each `$import` request based on `fhirServer/bulkdata/maxInputPerRequest`, which defaults to 5 input entries.
+
+The IBM FHIR Server supports an additional `storageDetail.type` with the value of `ibm-cos`.
+
+#### Example Request
+The following is a request to load data from the IBM COS endpoint into the IBM FHIR Server.
+
+``` sh
+curl -k -v -X POST -u "fhiruser:change-password" -H 'Content-Type: application/fhir+json' 'https://localhost:9443/fhir-server/api/v4/$import' -d '{
+    "resourceType": "Parameters",
+    "id": "30321130-5032-49fb-be54-9b8b82b2445a",
+    "parameter": [
+        {
+            "name": "inputFormat",
+            "valueString": "application/fhir+ndjson"
+        },
+        {
+            "name": "inputSource",
+            "valueUri": "https://localhost:9443/source-fhir-server"
+        },
+        {
+            "name": "input",
+            "part": [
+                {
+                    "name": "type",
+                    "valueString": "Patient"
+                },
+                {
+                    "name": "url",
+                    "valueUrl": "test-import.ndjson"
+                }
+            ]
+        },
+        {
+            "name": "storageDetail",
+            "valueString": "ibm-cos"
+        }
+    ]
+}'
+```
+
+#### Example Response
+The response body is populated only on error, the response status code is 202 and the `content-location` is populated with the polling location, such as the response header:
+
+``` sh
+content-location: https://localhost:9443/fhir-server/api/v4/$bulkdata-status?job=JKyViJ5Y0zcqQ2Uv1aBSMw%3D%3D
```

Comment:
Should be case-insenstivite, but I think I usually see it like `Content-Location`